app/model/Services/TemplateFactory.php

```diff
@@ -10,8 +10,8 @@
 
 class TemplateFactory
 {
-    public const SMTP_CREDENTIALS_ADDED = __DIR__ . '/../emails/smtpAdded.latte';
-    public const PAYMENT_DETAILS        = __DIR__ . '/../emails/payment.base.latte';
+    public const OAUTH_ADDED     = __DIR__ . '/../emails/oAuthAdded.latte';
```

Comment:
Jestli koukám správně, tak teď už se při přidání nového emailu neposílá potvrzovací email na adresu uživatele, který jej přidal. To asi dává smysl, protože nepotřebujeme otestovat funkčnost SMTP přístupů, každopádně v tom případě je potřeba dropnout tuhle šablonu.

Reply:
ok

Reply:
Tohle furt není dropnuto.
Zůstává nám tam ten soubor `oAuthAdded.latte`, který se ale nikde nepoužívá.

Reply:
už to tam není